ide/provisioning/install-debian-packages.sh

```diff
@@ -44,11 +44,12 @@ apt-get install $APTOPTS make g++ \
   imagemagick gettext \
   mesa-common-dev libgl1-mesa-dev libegl1-mesa-dev \
   fonts-dejavu \
-  libnetcdf-dev
+  libnetcdf-dev \
+  xz-utils
 echo
 
 echo Installing dependencies for compiling with LLVM / Clang...
-apt-get install $APTOPTS llvm clang-4.0 libc++-dev
+apt-get install $APTOPTS llvm clang libc++-dev
```

Comment:
Oh, now we're back to plain unversioned `clang`. Why submit a PR changing it forth and back?

Reply:
See https://github.com/XCSoar/XCSoar/pull/241#discussion_r314697580. Removed.